frost-ed25519/src/keys/repairable.rs

```diff
@@ -1,11 +1,10 @@
-/// Repairable Threshold Schemes
-#![doc = include_str!("../../repairable.md")]
```

Comment:
Do we not want to generate docs for this? These have been removed in this commit

Reply:
I think that was confusing a comment...

After I saw your comment I copied those docs, so it should look better now